### ISSUE TITLE
[Bug]: Adding a backslash `\` in Create new civillization panel for planet name, internal server error 500

### ISSUE THREAD

**Issue by Willmish** [BillmanH/exoplanets#72]
To reproduce:
When Creating a new civillization, add a backslash `\` for the planet name (I typed in 'D\/PA'). Will cause an internal 500 server error

**Comment by BillmanH** [comment]
I was able to create `D/PA` in the dev Tenant: 
![image](https://user-images.githubusercontent.com/10108711/208262824-50238cc9-6170-48f2-9f4f-e9b3acbb122a.png)
However, `D\PA` causes `GraphSyntaxException`:
```
Gremlin query syntax error: Unexpected token: '''; in input: 'g.addV('planet').property('name',''. @ line 1, column 34.
```
Looking into the request made by in `models.py` the request looks properly escaped:
```
"g.addV('planet').property('name','D\\PA')
  .property('class','terrestrial')
  .property('objid','2875301669077')
  .property('label','planet')
  .property('radius',0.0814)
  .property('mass',0.9808)
  .property('orbitsDistance',2)
  .property('orbitsId','9480234285386')
  .property('orbitsName','')
  .property('isSupportsLife','True')
  .property('isPopulated','True')
  .property('isHomeworld','True')
  .property('username','BillmanLocal2')
  .property('objtype','planet')"
``
Interestingly, running it in the portal works just fine: 
![image](https://user-images.githubusercontent.com/10108711/208263486-c7eecbc3-00f0-4fd4-9b8f-348510dc4079.png)

I'll open an SO ticket to see if anyone else has encountered the issue, but for now, I think I'll just clean out the string by adding the following to `CosmosdbClient.cs()` **c**lean **s**tring:
```python
s = str(s).replace("\\","-")
```
It's not great. But it will fix the issue until I'm able to get a solution from the Azure CosmosDB team.  

![image](https://user-images.githubusercontent.com/10108711/208264067-86109811-88c6-425d-96d4-18c7fa9464ad.png)

This is done in `dev` and I'll include it in the next release.

**Comment by BillmanH** [comment]
[SO post is here](https://stackoverflow.com/questions/74837169/azure-cosmosdb-query-runs-fine-in-portal-but-not-in-text-string) just in case it comes up later

**Comment by BillmanH** [comment]
fixed in #73

**Comment by Willmish** [comment]
Nice one on the quick fix, curious to see why the behaviour is ambigous.
